virt/learn_more_about_openshift_virtualization.adoc

```diff
@@ -0,0 +1,108 @@
+include::modules/virt-document-attributes.adoc[]
+[id="learn_more_about_openshift_virtualization"]
+= Learn more about {VirtProductName}
```

Comment:
too generic, and does not explain what OpenShift Virtualization means.  "Learn more about deploying and managing virtual machine on your openshift cluster. (or something like that)

Reply:
Changed to Learn more about deploying and managing virtual machines in your {product-title} cluster.